ansible/inventory/genestack/group_vars/k8s_cluster/addons.yml

```diff
@@ -179,30 +179,30 @@ metallb_speaker_enabled: "{{ metallb_enabled }}"
 # metallb_protocol: "layer2"
 # metallb_port: "7472"
 # metallb_memberlist_port: "7946"
-metallb_config:
```

Comment:
WE can leave all this like it was, these are example vars and have no effect.

Reply:
Rolled it back